### ISSUE TITLE
Inconsistent Reported Results

### ISSUE THREAD

**Issue by russ-1234** [VISION-SJTU/RSD#3]
<img width="810" height="336" alt="Image" src="https://github.com/user-attachments/assets/31ba66bf-9303-4984-9c3a-c49e19aa8a4c" />

In the main Table 1, the result is 78.50.  Which one is correct.

**Comment by spacewanderx** [comment]
Hi @russ-1234 , thanks for your interest in our work. 

The difference in values is because this specific part of this ablation was done using a configuration  (e.g., number of GPUs, batch size, and learning rate) different from the one for main experiments, due to limited computing resources available at that time. We did not tune the hyperparameters for this ablation study, so the result can be different.

**Comment by russ-1234** [comment]
> Hi [@russ-1234](https://github.com/russ-1234) , thanks for your interest in our work.
> 
> The difference in values is because this specific part of this ablation was done using a configuration (e.g., number of GPUs, batch size, and learning rate) different from the one for main experiments, due to limited computing resources available at that time. We did not tune the hyperparameters for this ablation study, so the result can be different.

Thanks for the quick reply.  Can you nicely provide the exact configuration, including the number of GPUs, Bs, lr, etc., for reproducing the results in Table 1.

**Comment by spacewanderx** [comment]
Hi, for all experiments in Table 1, we used 2 GPUs. The per-GPU batch size and the learning rate follow the configs in configs/cifar/cnn.yaml and configs/cifar/vit_mlp.yaml. We used the command python -m torch.distributed.launch --nproc_per_node=2.
